cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/AbstractConductingEquipmentConversion.java

```diff
@@ -372,6 +372,14 @@ protected static EquivalentInjectionConversion getEquivalentInjectionConversionF
         }
     }
 
+    // TODO graph is only valid if cgmes export preserves separated models
+    private EquivalentInjectionConversion getEquivalentInjectionConversionForDanglingLine(String graph, String boundaryNode) {
+        List<PropertyBag> eis = context.boundary().equivalentInjectionsAtNode(boundaryNode);
+
+        Optional<PropertyBag> optionalEi = eis.stream().filter(ei -> ei.get("graph").equals(graph)).findFirst();
```

Comment:
Please create a constant to replace the ```"graph"``` literal and comply with the sonar cloud check.

Reply:
Done